website/auth.py

```diff
@@ -366,6 +362,9 @@ def signup():
             return gettext('keyword_language_invalid'), 400
 
         # Validations, optional fields
+        if 'email' in body:
+            if not isinstance(body.get('email'), str) or not utils.valid_email(body.get('email')):
+                return gettext('email_invalid')
```

Comment:
This line needs to have `, 400` appended to it. Otherwise, the user does not get a proper error when they input an invalid email, but are happily redirected to the /landing-page without being logged it.

Reply:
Good catch!